_episodes/01-run-quit.md

```diff
@@ -5,55 +5,153 @@ exercises: 0
-*   It also installs all the extra libraries it needs to run.
-*   Once you have installed Python and the Jupyter Notebook requirements, open a shell and type:
+JupyterLab is included as part of the the Anaconda Python distribution. If you have not already 
+installed the Anaconda Python distribution, see [the setup instructions]({{ page.root }}/setup/) 
+for installation instructions.
 
-    ~~~
-    $ jupyter notebook
-    ~~~
+JupyterLab is an application whose frontend user interface runs in a web browser.
 
-*   This will start a Jupyter Notebook server and open your default web browser. 
 *   The server runs locally on your machine only and does not use an internet connection.
-*   The server sends messages to your browser.
-*   The server does the work and the web browser renders the notebook.
-*   You can type code into the browser and see the result when the web page talks to the server.
-*   This has several advantages:
-    *   You can easily type, edit, and copy and paste blocks of code.
-    *   Tab complete allows you to easily access the names of things you are using
-        and learn more about them.
-    *   It allows you to annotate your code with links, different sized text, bullets, etc.
-        to make it more accessible to you and your collaborators.
-    *   It allows you to display figures next to the code that produces them
-        to tell a complete story of the analysis.
+*   The server sends messages to your web browser.
+*   The server does the work and the web browser renders the result.
+*   You will type code into the browser and see the result when the web page talks to the server.
+
+## Starting JupyterLab
+
+### Mac OS X
+To start the JupyterLab server you will need to access the command line through the Terminal. 
+There are two ways to open Terminal on Mac.
+
+1. In your Applications folder, open Utilities and double-click on Terminal
+2. Press <kbd>Command</kbd> + <kbd>spacebar</kbd> to launch Spotlight. Type `Terminal` and then 
+double-click the search result or hit <kbd>Enter</kbd>
+
+After you have launched Terminal, type the command to launch the JupyerLab server.
+
+~~~
+$ jupyter lab
+~~~
+{: .bash}
+
+### Windows Users
+To start the JupyterLab server you will need to access the command line through the Command 
+Prompt. Use the shortcut keys <kbd>Windows Logo Key</kbd> + <kbd>R</kbd> to launch the Run window, 
+then type in `cmd` and press Enter.
+
+After you have launched the Command Prompt, type the command:
+
+~~~
+$ jupyter lab
+~~~
+{: .bash}
+
+Below is a screenshot of a similar JupyterLab landing page to the one that should open in your 
+default web browser after starting the JupyterLab server on wither Mac OS X or Windows.
+
+<center>
+    <img alt="JupyterLab landing page" src="../fig/0_jupyterlab_landing_page.png" width="750"/>
+</center>
+
+## The JupyterLab Interface
+
+JupyterLab provides flexible building blocks for interactive, exploratory computing. While 
```

Comment:
Minor nitpick: how about simplifying this paragraph to 

> JupyterLab has many features found in traditional integrated development environments (IDEs) but is focused on providing flexible building blocks for interactive, exploratory computing.

instead?

Reply:
Works for me!